### ISSUE TITLE
[SUGGESTION]: Quest list completion multiplier

### ISSUE THREAD

**Issue by Robbers-Daughter** [pokeclicker/pokeclicker#5130]
### Current Behavior

Upon completing all quests on your quest list you get a free refresh and nothing else. The refresh becomes very affordable quite early in the game so it kind of stops being a reward.

### Improved Behavior

Give players a bonus multiplier on quest points from the next set of quests if you complete a full list.

### Reason for change

Currently a lot of players simply do quests that are worth a lot of quest points that are easy/fast to do and just pay for a refresh to maximize quest point gain as fast as possible. The reason for this change is to reward players for doing all quests even if they feel arduos and discourage refresh spamming.

### Detailed Description

The idea is that upon completing every quest of your quest list, in addition to the free refresh, you also gain a multiplier on every quest on your next set of quests. So for example if the next set of quests have a quest for 1000qp it would be worth 1500qp with the multiplier (While I personally think 1.5 multiplier is good, the amount could be something else if you think it's too high or low). The multiplier would obviously be lost if you click refresh until you complete a full list again.

**Comment by Askellown** [comment]
this would be great if the "harvest x <berry type> couldn't drop 1h+ berries. as it is, it's not worth it regrdless of how high the multi is imo

**Comment by Robbers-Daughter** [comment]
> this would be great if the "harvest x couldn't drop 1h+ berries. as it is, it's not worth it regrdless of how high the multi is imo

I agree that most players would probably not bother with those quests even for a multiplier as a reward. However I would still consider it better for there to be some incentive to at least consider not refreshing early. You won't get those quests every time and you'd just lose the multiplier for one round.

With that said even with a multiplier you might still get better qp/h by spamming refresh but at least the people who do all quests would get a reward for doing so.

**Comment by Robbers-Daughter** [comment]
The lack of reward for full quest list completion was the main reason for my suggestion. With #5599 there is now a reward for that so I believe I can close this.
